Add CNES salary tables

ivalaginja opened the pull request:
Adding CNES salary tables to the website

---

For more details, open the [Copilot Workspace session](https://copilot-workspace.githubnext.com/adigioacchino/CNRSPostdocSalary?shareId=1a179cdc-f25e-43ed-8461-79650d95ee18).

adigioacchino commented:
Hi Iva!
Thank you for your help!
I added a readme with instructions on how to serve the website locally so you can see if your changes are what you intended them to be. I also changed a bit the code that prepares the table so that CNES tables do not have replicate rows.

Now, before I merge, do you want to add a sentence about your sources or add your contact info to the page so that interested people can contact you directly? We should do this, WDYT @MauroPastore?

ivalaginja commented:
@adigioacchino I added my contact info and clarified the concrete sources. I also added a missing link to the new tables on the home page.
When I serve the website locally though, I still see three columns in the CNES data, is this intended?

adigioacchino commented:
Yes it's useful to know that CNES salaries do not depend on where the postdoc is based.

I merged your changes! 🚀
